modules/codegen-plugin/src/sbt-test/codegen-plugin/multimodule-staged/foo/src/main/smithy/foo.smithy

```diff
@@ -0,0 +1,12 @@
+$version: "2.0"
+
+namespace foo
+
+use smithy4s.api#uuidFormat
```

Comment:
instead of a fully fledged test, can we reuse the ones in `modules/codegen-plugin/src/sbt-test/codegen-plugin/multimodule/` ? the change is minimal and that would avoid one extra sbt-test